Documentation/boards/README.md

```diff
@@ -39,17 +40,17 @@ Notes:
 
 |Board|Build|Config|Docs|
 |-----|----|------|----|
-|Pi4B 32-bit    |`make rpi4`           |[rpi4](../../buildroot-external/configs/rpi4_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi4B 64-bit    |`make rpi4_64`        |[rpi4_64](../../buildroot-external/configs/rpi4_64_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi3B 32-bit    |`make rpi3`           |[rpi3](../../buildroot-external/configs/rpi3_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi3B 64-bit    |`make rpi3_64`        |[rpi3_64](../../buildroot-external/configs/rpi3_64_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi2            |`make rpi2`           |[rpi2](../../buildroot-external/configs/rpi2_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi Zero        |`make rpi0_w`         |[rpi0_w](../../buildroot-external/configs/rpi0_w_defconfig)|[raspberrypi](./raspberrypi/)|
-|Pi             |`make rpi`            |[rpi](../../buildroot-external/configs/rpi_defconfig)|[raspberrypi](./raspberrypi/)|
-|Odroid-C2      |`make odroid_c2`      |[odroid_c2](../../buildroot-external/configs/odroid_c2_defconfig)|[hardkernel](./hardkernel/)|
-|Odroid-C4      |`make odroid_c4`      |[odroid_c4](../../buildroot-external/configs/odroid_c4_defconfig)|[hardkernel](./hardkernel/)|
-|Odroid-N2      |`make odroid_n2`      |[odroid_n2](../../buildroot-external/configs/odroid_n2_defconfig)|[hardkernel](./hardkernel/)|
-|Odroid-XU4     |`make odroid_xu4`     |[odroid_xu4](../../buildroot-external/configs/odroid_xu4_defconfig)|[hardkernel](./hardkernel/)|
-|Tinker Board   |`make tinker`         |[tinker](../../buildroot-external/configs/tinker_defconfig)|[asus](./asus/)|
-|Generic x86-64 |`make generic_x86_64` |[generic_x86_64](../../buildroot-external/configs/generic_x86_64_defconfig)|[generic-x86-64](./generic-x86-64/)|
-|OVA            |`make ova`            |[ova](../../buildroot-external/configs/ova_defconfig)|[ova](./ova/)|
+|Pi4B 64-bit   |`make rpi4_64`       |[rpi4_64](../../buildroot-external/configs/rpi4_64_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi4B 32-bit   |`make rpi4`          |[rpi4](../../buildroot-external/configs/rpi4_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi3B 64-bit   |`make rpi3_64`       |[rpi3_64](../../buildroot-external/configs/rpi3_64_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi3B 32-bit   |`make rpi3`          |[rpi3](../../buildroot-external/configs/rpi3_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi2           |`make rpi2`          |[rpi2](../../buildroot-external/configs/rpi2_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi Zero       |`make rpi0_w`        |[rpi0_w](../../buildroot-external/configs/rpi0_w_defconfig)|[raspberrypi](./raspberrypi/)|
+|Pi            |`make rpi`           |[rpi](../../buildroot-external/configs/rpi_defconfig)|[raspberrypi](./raspberrypi/)|
```

Comment:
Pi Zero and Pi configurations actually got recently dropped, we should remove them from this table too.

Reply:
Done. I removed Pi Zero and Pi from the tables. In addition I removed note for Raspberry Pi 4 (all RAM versions are supported so it's not needed). What about Raspberry Pi Compute Module 4? It's officially supported or not? No info about that on the [Installation docs](https://www.home-assistant.io/installation/raspberrypi).

Reply:
> What about Raspberry Pi Compute Module 4?

It should work with the rpi4 image and is generally supported. That said, we don't have instructions etc. Then also different IO board might need a different device tree...